README.md

```diff
@@ -36,7 +36,7 @@ npm i tic80-js --save
 
 3. Build and run the game through TIC-80 JS:
     ```
-    tic80-js index.js
+    npx tic80-js index.js
```

Comment:
`npx` ?

Reply:
https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b

As of npm 5.2.0 there is a new command, npx, which let's you execute npm modules without first installing them as globals.  If you haven't installed the module at all, it will go pull it out of npm or a GitHub gist for you, which is potentially dangerous.  But as long as you installed it already, npx will just fire off your local copy of the module.

I think this is the easiest way to using tic80-js.